app/views/admin/tags/show.html.erb

```diff
@@ -0,0 +1,30 @@
+<h1><%= @title = "Tags – #{@tag}" %></h1>
```

Comment:
Super minor, but let's make it "Tag –" as we're talking about a singular record.